test/integration/targets/filter_urls/jinja26.yml

```
@@ -0,0 +1,25 @@
+- hosts: testhost
```

Use localhost since no modules are involved (controller execution is all that is needed).

```suggestion
- hosts: localhost
```